python/ray/tune/tests/execution/test_controller_checkpointing_integration.py

```diff
@@ -409,118 +409,6 @@ def test_checkpoint_at_end_not_buffered(
         assert num_checkpoints(trial) == 1
 
 
-@pytest.mark.parametrize(
-    "resource_manager_cls", [FixedResourceManager, PlacementGroupResourceManager]
-)
-def test_checkpoint_user_checkpoint(
```

Comment:
Context for deleting these tests?

Reply:
it was testing some weird user-checkpointing frequency logic that was specific to the `__fake` trainable. Like a user defined checkpoint frequency with a class trainable.